src/Browsershot.php

```diff
@@ -489,6 +489,13 @@ public function bodyHtml(): string
         return $this->callBrowser($command);
     }
 
+    public function cookie(): string
```

Comment:
Maybe we should rename this to `getCookies` so we avoid that people might think this can be used to set cookies.

Reply:
would prefer getCookies too, already had it that way the first time i implemented it on my side ... but as the other Output Options are also just called by Name, not get or something like that I chanced it accordingly. Sure on this to start new "naming schema"?